CHANGELOG.md

```diff
@@ -4,6 +4,83 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.48.0
+
+### Important Changes
+
+- **feat(core)**: Make `runWithAsyncContext` public API (#7817)
+- **feat(node)**: Auto-select best `AsyncContextStrategy` for Node.js version (#7804)
+
+This release introduces an new API to the Node SDK: `runWithAsyncContext` can be used to isolate Sentry scope and breadcrumbs to a single request if you are finding that breadcrumbs and scope are leaking across requests.
+With this new API, the SDK will use `AsyncLocalStorage` in favour of domains if you're using Node 14 or newer. It still falls back to using domains on older Node versions.
+
+Here's a usage example for [using Sentry in a Koa project](https://docs.sentry.io/platforms/node/guides/koa/#monitor-performance), where previously, you needed to use domains:
+
+```js
+const requestHandler = (ctx, next) => {
+  return new Promise((resolve, reject) => {
+    Sentry.runWithAsyncContext(
+      async hub => {
+        hub.configureScope(scope =>
+          scope.addEventProcessor(event =>
+            Sentry.addRequestDataToEvent(event, ctx.request, {
+              include: {
+                user: false,
+              },
+            })
+          )
+        );
+
+        await next();
+        resolve();
+      },
+      { emitters: [ctx] }
+    );
+  });
+};
+```
+
+If you're manually using domains to isolate Sentry data, we strongly recommend switching to this API!
+
+In addition to exporting `runWithAsyncContext` publicly, the SDK also uses it internally where we previously used domains.
+
+- **feat(sveltekit)**: Remove `withSentryViteConfig` (#7789)
+
+This release removes our `withSentryViteConfig` wrapper we previously instructed you to add to your `vite.config.js` file. It is replaced Vite plugins which you simply add to your Vite config, just like the `sveltekit()` Vite plugins. We believe this is a more transparent and Vite/SvelteKit-native way of applying build time modifications. Here's how to use the plugins:
+
+```js
+// vite.config.js
+import { sveltekit } from '@sveltejs/kit/vite';
+import { sentrySvelteKit } from '@sentry/sveltekit';
+
+export default {
+  plugins: [sentrySvelteKit(), sveltekit()],
+  // ... rest of your Vite config
+};
+```
+
+Please note that this is a **breaking change**! We're sorry for the inconvenience but the SvelteKit SDK is still in alpha stage and we want to establish a clean and SvelteKit-friendly API before making the SDK stable. You have been warned ;)
```

Comment:
Should we link to SvelteKit readme here?

Reply:
Sure, can't hurt!